Documentation/PackedAssetBuilds.md

```diff
@@ -1,12 +1,40 @@
 # Packaged MultiplayerSample Builds
+
 To make relocatable client and server builds for the MultiplayerSample, we recommend making packaged builds. These package builds will contain the Game or Server Launcher and the bundled assets needed to run the launcher outside of the developer environment.
 
 You can make both release packaged builds or profile packaged builds. For more information about creating release builds, see the O3DE documentation on [Creating a Project Game Release Layout for Windows](https://www.o3de.org/docs/user-guide/packaging/windows-release-builds/).
 
 The guide below covers how to make profile packaged builds which are very useful for early sharing and play testing.
 
+## Brief outline of the packaging steps
+
+1. Install WWise SDK (required for o3de-multiplayersample but optional for other projects)
+1. Use the engine from GitHub, not the installer
+1. Create a build for non-monolithic build of the engine
+1. Compile the engine with o3de-multiplayersample in profile non-monolithic mode
+1. Process all the assets using the Asset Processor from profile non-monolithic mode
+1. Create game bundles (.pak files) from the seed lists (a list of game assets)
+1. Create a build for monolithic build of the engine
+1. Compile the engine with o3de-multiplayersample in profile (and/or release) monolithic mode
+1. Create a folder for the packaged build outside of the engine, this will be the share-able folder of the packaged game
+1. Copy the monolithic game binaries and the game bundles to the packaged folder
+1. The packaged build is ready and can be shared without requiring the developer environment.
```

Comment:
Possible suggestions to make this list shorter/simpler

```suggestion
1. Install WWise SDK (required for o3de-multiplayersample but optional for other projects)
1. Use the engine from GitHub, not the installer
1. Compile o3de-multiplayersample and the engine (non-monolithic)
1. Process assets using the Asset Processor
1. Create asset bundles (.pak files) from asset seed lists (a list of game assets)
1. Compile o3de-multiplayersample and the engine (monolithic profile or release)
1. Copy the game binaries and asset bundles to a release folder you can share
```

Reply:
OK